tests/codegen/align-fn.rs

```diff
@@ -116,3 +117,23 @@ pub fn align_specified_twice_2() {}
 #[align(32)]
 #[align(256)]
 pub fn align_specified_twice_3() {}
+
+const _: () = {
+    // CHECK-LABEL: align_unmangled
+    // CHECK-SAME: align 256
+    #[unsafe(no_mangle)]
+    #[align(32)]
+    #[align(256)]
+    extern "C" fn align_unmangled() {}
```

Comment:
Weren't we going to warn on applying `align` twice?

I mean that shouldn't affect the codegen test, I'm just realizing that I don't know for sure if we have a test that does check that we warn.

Reply:
It would be nice to have a warning, but it’s not required for the MVP. It hasn’t been implemented yet.